### ISSUE TITLE
Documentation for the functions and attributes

### ISSUE THREAD

**Issue by Jopepato** [AnthonyMRios/leml#1]
Hi Anthony!
Thanks for this algorithm and for the implementation, I really like your paper. Right now I'm working with some multi-label models so I was trying yours as weel and I think that it miss some documentation in the code. Such as in the functions of fit and predict, as well as the parameters for the model.
Thank you very much again for the code!

**Comment by AnthonyMRios** [comment]
Hi Jose,

I can add more documentation when I have some time. Please go through the example code for more information about how to run it. The fit/predict functions should follow the same scheme as scikit-learn. Everything else is specific to the optimization procedures. If you have a specific error, I'm happy to help out. For the parallel version, I think you need to set the "OMP_NUM_THREADS" environment variable, but I have to check to be 100% sure.

By the way, the paper is not mine, but I also found it interesting. I implemented it because I wanted to apply it to my multi-label problems and I did not want to use the matlab version of the original paper.

**Comment by Jopepato** [comment]
Thank you very much for your response! I looked at the example yesterday and made it work! Thank you for the code, truly.
The thing is that in the example you load a python object, so I was a bit confused on what should expect fit, as X and y, were not in plain text and extracted from an object.
But looking at the other functions I managed it. I can make a PR making some documentation for the functions as well as the parameters that it uses ^^
